lib/src/test/kotlin/starknet/utils/DevnetClient.kt

```diff
@@ -56,7 +56,7 @@ class DevnetClient(
 
     companion object {
         // Source: https://github.com/0xSpaceShard/starknet-devnet-rs/blob/47ee2a73c227ee356f344ce94e5f61871299be80/crates/starknet-devnet-core/src/constants.rs
-        val accountContractClassHash = Felt.fromHex("0x61dac032f228abef9c6626f995015233097ae253a7f72d68552db02f2971b8f")
+        val accountContractClassHash = Felt.fromHex("0x044cab2e6a3a7bc516425d06d76c6ffd56ae308864dbc66f8e75028e3784aa29")
```

Comment:
note: Class hash of predeployed OZ accounts has changed.

<img width="878" alt="image" src="https://github.com/user-attachments/assets/5de013ef-c96d-4549-ad61-2e3858fd625e" />